crates/sui-indexer/src/models/system_state.rs

```diff
@@ -51,11 +51,11 @@ impl From<SuiSystemStateSummary> for DBSystemStateSummary {
             reference_gas_price: s.reference_gas_price as i64,
             safe_mode: s.safe_mode,
             epoch_start_timestamp_ms: s.epoch_start_timestamp_ms as i64,
-            governance_start_epoch: s.governance_start_epoch as i64,
+            stake_subsidy_start_epoch: s.stake_subsidy_start_epoch as i64,
             epoch_duration_ms: s.epoch_duration_ms as i64,
-            stake_subsidy_epoch_counter: s.stake_subsidy_epoch_counter as i64,
+            stake_subsidy_epoch_counter: s.stake_subsidy_distribution_counter as i64,
```

Comment:
nit: do we want to change the field names here too?

Reply:
Happy to do that, maybe as a follow up?